### PR TITLE
Features/select all

### PR DESCRIPTION
**Pull request by felippepuhle**

### CODE REVIEW

**Comment by sibelius** [comment]
can we avoid redux here? this won't scale

imagine N datalists where I would like to select a few items per list, how can I do that?

**Comment by felippepuhle** [comment]
I used redux aiming connect one or more selection list...

We could remove from here and just call some callback function, and we can use redux on the real app if needed...

What do you think?

**Comment by felippepuhle** [comment]
Note: using redux on the 'real app' will force us to do something like this:

```js
<DataList
    table={{
      checkboxes: {
        preselectedValues: { object1, object3, etc },
        component: (props) => <input type="checkbox" {...props} />
      },
    }}
/>
```

Couldn't think a better solution for now... 🤔

File: src/components/DataContainer.js [hunk 1]
```diff
@@ -47,22 +48,60 @@ class DataContainer extends Component {
     }
   }
 
-  render() {
-    const { table } = this.props;
-    const { loading } = this.state;
+  check(item) {
+    const checked = [...this.state.checked];
+    checked.push(item);
```
Comment (sibelius) [inline src/components/DataContainer.js]: `const checked = [...this.state.checked, item]`

File: src/components/DataContainer.js [hunk 2]
```diff
@@ -47,22 +48,60 @@ class DataContainer extends Component {
     }
   }
 
-  render() {
-    const { table } = this.props;
-    const { loading } = this.state;
+  check(item) {
```
Comment (sibelius) [inline src/components/DataContainer.js]: `check = (item) =>` ?

File: src/utils/props.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+// @flow
+export const sanitizeVariables = (variables: Object): Object => {
+  const sanitized = { ...variables };
+  if (!sanitized.first) {
+    sanitized.first = 20;
+  }
```
Comment (sibelius) [inline src/utils/props.js]: `return {
   first: 20,
   ...variables,
}`

**Review by sibelius — APPROVED** [review]

**Pull request merged**